engine/src/main/java/org/destinationsol/drawable/systems/GraphicsRenderer.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.drawable.systems;
+
+import com.badlogic.gdx.math.Vector2;
+import org.destinationsol.common.In;
+import org.destinationsol.drawable.GraphicsElement;
+import org.destinationsol.drawable.components.Graphics;
+import org.destinationsol.drawable.components.Invisibility;
+import org.destinationsol.drawable.events.RenderEvent;
+import org.destinationsol.entitysystem.EntitySystemManager;
+import org.destinationsol.entitysystem.EventReceiver;
+import org.destinationsol.game.GameDrawer;
+import org.destinationsol.location.components.Angle;
+import org.destinationsol.location.components.Position;
+import org.terasology.gestalt.entitysystem.entity.EntityRef;
+import org.terasology.gestalt.entitysystem.event.EventResult;
+import org.terasology.gestalt.entitysystem.event.ReceiveEvent;
+
+/**
+ * This handles the drawing of each entity with a {@link Graphics} component.
+ */
+public class GraphicsRenderer implements EventReceiver {
```

Comment:
I still need to learn what makes the distinction in the newest Gestalt of an even receiver vs an entity system class. So used to everything with logic being a `System` leading to easy names like "RenderingSystem"

Maybe leaving `Component` off from actual component classes leads to some benefits, but others like event classes still seem to end in `-Event` - I wonder if it would make sense to keep a focus on `System` classes too.  I just really don't know why event handlers and systems are two different things now 🤔 

This isn't on you as much as me just not catching some subtleties or knowing about any sort of reasoning things were split out, yet. Just everything in my head screams that this is a system class :-)